components/Badge/__tests__/Badge.test.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-env jest */
+import React from 'react';
+import createSnapshotTest from 'test-utils/createSnapshotTest';
+
+import Badge from '../Badge';
+
+describe('Badge', () => {
+  it('should render with just required props passed', () => {
```

Comment:
I'm glad you used the `createComponent` script 😁 

Unfortunately, I changed it on you while you were working on this page.

Can you change both of the test names to:
`should render with required props`

and

`should render with many props assigned`

Also, please pass it the correct props so that the snapshot test properly reflects how the component would appear in the DOM.